install.sh

```diff
@@ -41,23 +42,29 @@ case "${1}" in
                 fi
 
                 # check if systemd services are installed
-                if [ -e ${SYSTEMD_DIR}/adaway-linux.timer ] || [ -e ${SYSTEMD_DIR}/adaway-linux.service ] ; then
+                if [ -e "${SYSTEMD_DIR}"/adaway-linux.timer ] || [ -e "${SYSTEMD_DIR}"/adaway-linux.service ] ; then
 
                   echo "[!] Removing systemd service..."
                   # Unhooking the systemd service
                   systemctl stop adaway-linux.timer && systemctl disable adaway-linux.timer || echo "[!] adaway-linux.timer is missing. Have you removed it?"
                   systemctl stop adaway-linux.service && systemctl disable adaway-linux.service || echo "[!] adaway-linux.service is missing. Have you removed it?"
-                  rm ${SYSTEMD_DIR}/adaway-linux.*
+                  rm "${SYSTEMD_DIR}"/adaway-linux.*
```

Comment:
here the same. Please use `"` for the complete file name.

Reply:
@sedrubal We can't put the * into " " as it'll lose it's purpose.